blocks-touch/b-slider/b-slider.js

```diff
@@ -127,6 +127,11 @@
             slider._currentX = slider._currentX || 0;
             // объект с текущими параметрами каждого слайда
             slider._touch = {};
+            // страница для отслеживания pointermove / pointerup
+            slider._page = $(window);
+            // pointer-хэндлеры привязаные к контексту исполнения
+            slider._bonPointerMove = slider._onPointerMove.bind(slider);
```

Comment:
It need for `off(event, handler)` (unsubscribe from events) because bind(fn) is new instance of function every call.
```javascript
var a = function() {}
a.bind(a) === a.bind(a)
// > false
```
So we must store bound handler to use for unsubscribing.